.github/workflows/publish_on_request.yml

```
@@ -0,0 +1,105 @@
+name: Publish on request
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  build:
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/publish'}}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+
+      - uses: actions/checkout@v3
+        if: success()
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+
+      - uses: actions/checkout@v3
+        if: success()
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          cache: 'npm'
+          cache-dependency-path: package-lock.json
+          node-version: 16
+      - name: Install npm dependencies
+        run: |
+          npm ci
```

The pipe is usefull to run several commands, if only one it could be removed 🙂
```suggestion
        run: npm ci
        
```